src/components/VAudioTrack/VWaveform.vue

```diff
@@ -468,9 +467,19 @@ export default defineComponent({
       }
     }
 
+    const handleSpacebar = () => {
+      emit('toggle-playback')
+    }
+
+    /**
+     * @param {KeyboardEvent} event
+     */
     const willBeHandled = (event) =>
-      ['ArrowLeft', 'ArrowRight', 'Home', 'End'].includes(event.key)
+      ['ArrowLeft', 'ArrowRight', 'Home', 'End', ' '].includes(event.key)
```

Comment:
```suggestion
			// ' ' A single space is the `key` value for a space bar event
      ['ArrowLeft', 'ArrowRight', 'Home', 'End', ' '].includes(event.key)
```

something like this?